components/02-molecules/registration/ENSResolverComponent.tsx

```diff
@@ -182,8 +186,35 @@ export const ENSResolverComponent = ({
           </button>
         </div>
       </div>
+      {ensResolver === EnsResolver.Custom && (
+        <div className="flex w-full">
+          <Input
+            clearable
+            label={"Ens Resolver"}
+            placeholder={"Resolver address"}
+            type="text"
+            className="text-left"
+            value={customAddress}
+            onChange={(e) => {
+              setCustomAddress(e.target.value);
+            }}
+          />
+        </div>
+      )}
       <div className="w-full flex">
-        <NextButton disabled={ensResolver === null} onClick={handleNextStep} />
+        <NextButton
+          disabled={ensResolver === null}
+          onClick={() => {
+            if (ensResolver === EnsResolver.Custom) {
+              if (isAddress(customAddress)) {
+                setCustomResolverAddress(customAddress);
+                handleNextStep();
+              }
```

Comment:
Is an error displayed if no address is informed?

Reply:
The Button is disabled